media/js/base/mozilla-smoothscroll.js

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+ Portions based on http://blog.gospodarets.com/native_smooth_scrolling/
+*/
+
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+Mozilla.smoothScroll = (function() {
+    'use strict';
+
+    var _smoothScroll;
+
+    var _init = function() {
+        // try to use native smooth scrolling
+        if ('scrollBehavior' in document.documentElement.style) {
+            _smoothScroll = function(opts) {
+                // always smooth
+                opts.behavior = 'smooth';
+                window.scrollTo(opts);
+            };
+        // use jQuery if it's available
+        } else if (window.jQuery) {
+            var $htmlBody = $('html, body');
+
+            _smoothScroll = function(opts) {
+                $htmlBody.animate({
+                    scrollTop: opts.top,
+                    scrollLeft: opts.left
```

Comment:
When passing a single value for `opts.top` like in the example here, would it better to fall back to a `0` value for `opts.left` here instead of passing `undefined`, and vice versa?